Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,27 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
 # final image needs kubectl, so we copy `oc` from cli image to use as kubectl.
-FROM openshift/origin-cli:latest as cli
+FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
 # the base image is the ansible-operator's origin images
-FROM quay.io/openshift/origin-ansible-operator:4.6
+FROM registry.svc.ci.openshift.org/ocp/4.6:ansible-operator
```

Comment:
As compared to: `FROM openshift/ose-ansible-operator:4.6` for base in Dockerfile.rhel

Reply:
same here: are you just setting this for local dev/building? during CI builds this value should be getting replaced (if it's not, we should look into why not)

Reply:
so this image is being substituted by this logic:

https://github.com/openshift/release/blob/master/ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml#L21

https://github.com/openshift/release/blob/master/ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml#L2-L6

so changing this FROM line doesn't affect what CI is going to do.  It may make it clearer what image is being used (which is fine) but i wouldn't expect it to change the behavior/content.

(last FROM statement in the dockerfile is substituted)

Reply:
Ah, right @timflannagan1 mentioned that substitution early on and I forgot. But how does that work? I know from experience that is the URL is wrong (registry.svc.ci.openshift.org/ocp/4.6) the build will fail.

Reply:
so there are two substitutions that take place:

1) the last FROM in your dockerfile gets substituted by whatever this `from` references/resolves to:
https://github.com/openshift/release/blob/master/ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml#L21

2) earlier FROMs get substituted by this, if defined:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-master.yaml#L50

where in that example the "bin" tag (whatever pullspec it resolves to) is used to substitute anywhere in your dockerfile that `registry.svc.ci.openshift.org/ocp/builder:golang-1.12` appears.  you can reference any tag (one defined by your release config, or one that you know will exist because some other component builds it).

Reply:
@bentito you might find this a useful background: https://docs.google.com/presentation/d/1IVxsmB66fxRSo6q_C09DLkm7efth00puM7BCtjJJ3Po/edit#slide=id.g6bd699cf61_2_156

Reply:
full presentation here https://bluejeans.com/playback/s/ONXUWzK42mTUbtWR8IUbi8FEdf2Su0vOXdonC9AnSNAZFhppbZPxj1vdrynwigH0

Reply:
yep, now adding a layer for the existing (and needed for the boto packages) 4.4/base and COPYing from it.